src/operations/create_collection.ts

```diff
@@ -10,18 +10,19 @@ import type { PkFactory } from '../mongo_client';
 // eslint-disable-next-line
 import type { Collection } from '../collection';
 
-const ILLEGAL_COMMAND_FIELDS = new Set([
-  'w',
-  'wtimeout',
-  'j',
-  'fsync',
+const optionsAllowList = new Set([
```

Comment:
```suggestion
const OPTIONS_ALLOW_LIST = new Set([
```

I think our convention is to format top-level constants in uppercased snake case.